packages/nimble-components/src/anchor/styles.ts

```diff
@@ -2,102 +2,93 @@ import { css } from '@microsoft/fast-element';
-        outline: none;
-        box-shadow: inset 0px -1px ${linkFontColor};
-    }
-
-    .control:active {
-        color: ${linkActiveFontColor};
-        text-decoration: underline;
-    }
+    @layer base {
+        ${display('inline')}
 
-    .control${focusVisible}:active {
-        box-shadow: inset 0px -1px ${linkActiveFontColor};
-    }
+        :host {
+            box-sizing: border-box;
+            font: ${linkFont};
+        }
 
-    .control:not(:any-link) {
-        color: ${linkDisabledFontColor};
-        text-decoration: none;
-    }
+        .top-container {
+            display: contents;
+        }
 
-    .control:not(:any-link):active {
-        color: ${linkActiveDisabledFontColor};
-    }
+        [part='start'] {
+            display: none;
+        }
 
-    :host([underline-hidden]) .control {
-        text-decoration: none;
-    }
+        .control {
+            color: ${linkFontColor};
+            text-decoration: underline;
+        }
 
-    :host([underline-hidden]) .control:hover {
-        text-decoration: underline;
-    }
+        :host([underline-hidden]) .control {
+            text-decoration: none;
+        }
 
-    :host([underline-hidden]) .control${focusVisible} {
-        text-decoration: underline;
-    }
+        :host([appearance='prominent']) .control {
+            color: ${linkProminentFontColor};
+        }
 
-    :host([underline-hidden]) .control:not(:any-link) {
-        text-decoration: none;
-    }
+        .content {
+            pointer-events: none;
+        }
 
-    :host([appearance='prominent']) .control {
-        color: ${linkProminentFontColor};
+        [part='end'] {
+            display: none;
+        }
     }
 
-    :host([appearance='prominent']) .control:active {
-        color: ${linkActiveProminentFontColor};
+    @layer hover {
+        .control:any-link:hover {
+            text-decoration: underline;
+        }
     }
 
-    :host([appearance='prominent']) .control${focusVisible} {
-        box-shadow: inset 0px -1px ${linkProminentFontColor};
+    @layer focusVisible {
+        .control${focusVisible} {
+            outline: none;
+            box-shadow: inset 0px -1px;
+            text-decoration: underline;
+        }
     }
 
-    :host([appearance='prominent']) .control${focusVisible}:active {
-        box-shadow: inset 0px -1px ${linkActiveProminentFontColor};
-    }
+    @layer active {
+        .control:active {
+            color: ${linkActiveFontColor};
+            text-decoration: underline;
+            box-shadow: none;
+        }
 
-    :host([appearance='prominent']) .control:not(:any-link) {
-        color: ${linkProminentDisabledFontColor};
-    }
+        :host([appearance='prominent']) .control:active {
+            color: ${linkActiveProminentFontColor};
+        }
 
-    .content {
-        pointer-events: none;
+        ${
+            '' /* It's probably cleaner to add :not(active) to the selector in the 'active' layer (rather
```

Comment:
I don't think this comment is necessary. It is against nimble's guidelines to use the `:not` selector. Even if there wasn't a bug in the pseudo-states addon, I don't expect that we'd want to write the styles using `:not(active)`.

Reply:
If it worked, I would have argued for using `:not` here, since it more directly represents what we want, but that's moot. I'll just remove the comment.